frontend/public/components/utils/dropdown.jsx

```diff
@@ -467,41 +475,44 @@ class Dropdown_ extends DropdownMixin {
               </div>
             </button>
             {active && (
-              <ul
-                role="listbox"
-                ref={this.dropdownList}
-                className={classNames(
-                  'dropdown-menu__autocomplete-filter',
-                  'pf-c-dropdown__menu',
-                  menuClassName,
-                )}
-              >
-                {autocompleteFilter && (
-                  <div className="dropdown-menu__filter">
-                    <input
-                      autoFocus
-                      type="text"
-                      ref={(input) => (this.input = input)}
-                      onChange={this.changeTextFilter}
-                      placeholder={autocompletePlaceholder}
-                      value={autocompleteText || ''}
-                      autoCapitalize="none"
-                      onKeyDown={this.onKeyDown}
-                      className="pf-c-form-control"
-                      onClick={(e) => e.stopPropagation()}
-                      data-test-id="dropdown-text-filter"
-                    />
-                  </div>
-                )}
-                {this.renderActionItem()}
-                {bookMarkRows}
-                {_.size(bookMarkRows) ? (
-                  <li className="co-namespace-selector__divider">
-                    <div className="dropdown-menu__divider" />
-                  </li>
-                ) : null}
-                {rows}
-              </ul>
+              // Style the Application menu to match the Project selection menu
+              <div className="pf-c-menu pf-m-scrollable co-namespace-dropdown__menu">
+                <div className="pf-c-menu__content" style={{ maxHeight: '60vh' }}>
+                  {autocompleteFilter && (
+                    <>
+                      <div className="pf-c-menu__search">
+                        <input
+                          autoFocus
+                          type="text"
+                          ref={(input) => (this.input = input)}
+                          onChange={this.changeTextFilter}
+                          placeholder={autocompletePlaceholder}
+                          value={autocompleteText || ''}
+                          autoCapitalize="none"
+                          onKeyDown={this.onKeyDown}
+                          className="pf-c-form-control pf-m-search"
+                          onClick={(e) => e.stopPropagation()}
+                          data-test-id="dropdown-text-filter"
+                        />
+                      </div>
+                      <Divider />
```

Comment:
This is fine because its before the `ul`

```suggestion
                      <Divider />
```